cpp/src/aztec3/circuits/rollup/merge/.test.cpp

```diff
@@ -14,7 +15,49 @@ using DummyComposer = aztec3::utils::DummyComposer;
 } // namespace
 namespace aztec3::circuits::rollup::merge::native_merge_rollup_circuit {
 
-class merge_rollup_tests : public ::testing::Test {};
+class merge_rollup_tests : public ::testing::Test {
```

Comment:
I know you're just editing this, but I thought we were going for CamelCase classes